app/assets/stylesheets/application.scss

```diff
@@ -20,13 +20,13 @@
 @import "availabilities/index";
 @import "errors/allofthem";
 
-// @media screen and (max-width: 1060px) {
```

Comment:
Why did you uncomment all this?
In the master it's commented because it was causing bugs.